sirepo/numpy.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+u"""Wrappers for numpy
+
+:copyright: Copyright (c) 2021 RadiaSoft LLC.  All Rights Reserved.
+:license: http://www.apache.org/licenses/LICENSE-2.0.html
+"""
+from __future__ import absolute_import, division, print_function
+import numpy as np
+
+def ndarray_from_csv(path, skip_header, **kwargs):
+    def _read():
+        import csv
+        import re
+
+        with open(path) as f:
+            for r in csv.reader(f):
+                yield ','.join(map(lambda v: re.sub(r'"|\n|\r|,', '', v), r))
```

Comment:
The following is more traditional: `r'["\n\r,]'`

Is this guaranteed to work generally, that is, just stripping newlines and quotes?

Reply:
5ec2f51d .

I'm not exactly sure on the efficacy of the regex. @moellep was the [one to propose it](https://github.com/radiasoft/sirepo/issues/3123#issuecomment-725592205).  I know it solves the quoted values problem that we are trying to fix but I haven't seen any datasets with newlines in them.